include/deal.II/lac/la_parallel_vector.h

```diff
@@ -282,6 +282,15 @@ namespace LinearAlgebra
        */
       Vector(const Vector<Number, MemorySpace> &in_vector);
 
+      /**
+       * Move constructor. Uses the swap method.
+       *
+       * @note In order for this constructor to leave the moved-from object in a
+       * valid state it must allocate memory (in this case, an empty
+       * partitioner) - hence it cannot be marked as noexcept.
+       */
+      Vector(Vector<Number, MemorySpace> &&in_vector); // NOLINT
```

Comment:
This is not enough to make clang-tidy happy. I still think it makes sense to define a non-`noexcept` move constructor, so we need to turn that check off for this line. Is there some other syntax for this?

Reply:
I figured it out - I have to write `// NOLINT` at the end of the line containing the start of the ctor definition in the `.templates.h` file too.